doc/src/pair_sw.rst

```diff
@@ -14,7 +14,15 @@ Syntax
 
 .. code-block:: LAMMPS
 
-   pair_style sw
+   pair_style style keywords values
+
+* style = *sw* or *sw/gpu* or *sw/omp*
+* keyword = *modify*
+
+  .. parsed-literal::
+
+       *modify* value = flag
+         flag = 0/1 to turn off/on the modification
```

Comment:
When this functionality is implemented as a derived class, no keyword would be needed.

Reply:
Removed. See the new codes.

Reply:
Mind you with derived classes, it would also be trivial to add OPENMP package support (because you can do the same threebody() function replacement as with the MANYBODY version, just derive from the OPENMP pair style).

Reply:
Thanks for the suggestion. I added the sw/mod/omp style and the corresponding examples.

Reply:
There were multiple problems with the sw/mod/omp style: it was added to the wrong folder (MANYBODY instead of OPENMP), the /omp version was derived from sw/mod, but instead it should have been derived from sw/omp, which results in a much simpler and smaller (and more effective) implementation.